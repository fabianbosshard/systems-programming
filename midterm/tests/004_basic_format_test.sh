exec "$1" format=@b,@a delim=' ' <<EOF
Apple 5 3
Orange 10 7
Pear 6 2
EOF
