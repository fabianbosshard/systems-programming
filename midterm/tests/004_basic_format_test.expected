5,Apple
10,Orange
6,Pear
